jsapp/js/components/modalForms/translationSettings.es6

```diff
@@ -360,7 +360,7 @@ export class TranslationSettings extends React.Component {
                     {i !== 0 && (
                       <Button
                         type='bare'
-                        color='storm'
+                        color='dark-blue'
```

Comment:
Not here, but noticed a button alignment thing in the related languageForm.es6

![Screenshot_20240712_141325](https://github.com/user-attachments/assets/8f8ae296-6e35-4595-9d05-3b4a9661cee6)

Reply:
Fixed that too :)